src/script/connection/ConnectionRepository.ts

```diff
@@ -52,6 +52,7 @@ export class ConnectionRepository {
   private readonly connectionService: ConnectionService;
   private readonly userRepository: UserRepository;
   private readonly logger: Logger;
+  private deleteConnectionRequestConversation?: (userId: QualifiedId) => Promise<void>;
```

Comment:
We could probably find a naming that suggest it's a callback (here we feel it's going to trigger an imperative action)